modules/iam_users/outputs.tf

```diff
@@ -0,0 +1,10 @@
+# output for EKS module
+output "eks_iam_role_arns" {
+  value = [
+    for r in aws_iam_role.access_assumerole : {
+      arn  = r.arn
+      name = r.name
+    }
+  ]
+  description = "List of role and users"
```

Comment:
How about "List of mappings of users to roles"

Reply:
done